tests/fixtures/test_fixtures.py

```diff
@@ -72,6 +72,28 @@ def test_load_resource_types(app, db, es_clear):
     assert item_dict["props"]["datacite_general"] == "Collection"
 
 
+def test_load_community_types(app, db, es_clear):
+    id_ = 'communitytypes'
+    resource_types = GenericVocabularyEntry(
+        Path(__file__).parent / "data",
+        id_,
+        {
+            "pid-type": "comtyp",
+            "data-file": "vocabularies/community_types.yaml"
```

Comment:
might be we just need to specify the package to use defaults? (might need fixtures/data in the path)
```suggestion
            "data-file": "invenio_rdm_records/vocabularies/community_types.yaml"
```

Reply:
tried this change locally but tests fail when importing the fixtures.